<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>753</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>651</width>
     <height>691</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="customerTab">
    <attribute name="title">
     <string>고객용</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>280</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Click to Order</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>160</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Click to Order</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Click to Order</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>280</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Click to Order</string>
     </property>
    </widget>
    <widget class="QLabel" name="Menulabel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>38</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>27</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Cocktails</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>400</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Click to Order</string>
     </property>
    </widget>
    <widget class="QLabel" name="cocktail01">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>241</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="cocktail02">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>100</y>
       <width>241</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="cocktail03">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>220</y>
       <width>241</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="cocktail04">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>220</y>
       <width>241</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="cocktail05">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>340</y>
       <width>241</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="price01">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>231</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>price</string>
     </property>
    </widget>
    <widget class="QLabel" name="price02">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>130</y>
       <width>241</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>price</string>
     </property>
    </widget>
    <widget class="QLabel" name="price03">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>250</y>
       <width>241</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>price</string>
     </property>
    </widget>
    <widget class="QLabel" name="price04">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>250</y>
       <width>241</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>price</string>
     </property>
    </widget>
    <widget class="QLabel" name="price05">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>370</y>
       <width>241</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>price</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>490</y>
       <width>261</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="text">
      <string>cocktail name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>490</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>price</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>560</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="text">
      <string>카드를 터치하여 결제를 진행하세요</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="adminTab">
    <attribute name="title">
     <string>관리자용</string>
    </attribute>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>270</y>
       <width>121</width>
       <height>131</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>TURN ON</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>TURN OFF</string>
     </property>
    </widget>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>381</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>270</y>
       <width>121</width>
       <height>131</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_3">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>270</y>
       <width>121</width>
       <height>131</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_4">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>470</y>
       <width>121</width>
       <height>131</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_5">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>470</y>
       <width>121</width>
       <height>131</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_6">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>470</y>
       <width>121</width>
       <height>131</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="cup1">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>410</y>
       <width>111</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>66</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>고객 정보</string>
     </property>
    </widget>
    <widget class="QLabel" name="tank_1_warn">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>240</y>
       <width>66</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="tank_2_warn">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>240</y>
       <width>66</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="tank_3_warn">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>240</y>
       <width>66</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="tank_5_warn">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>440</y>
       <width>66</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="tank_4_warn">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>440</y>
       <width>66</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="tank_6_warn">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>440</y>
       <width>66</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="cup2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>410</y>
       <width>111</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="cup3">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>410</y>
       <width>111</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="cup6">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>610</y>
       <width>111</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="cup5">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>610</y>
       <width>111</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="cup4">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>610</y>
       <width>111</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="status">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>30</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
